syntax = "proto3";

package proto;

message SecuredFlag {
    bytes encrypted_flag = 1;
    bytes description = 2;
    int32 value = 3;
    fixed32 xor_key = 4;
}
